packages/neon/neon/lib/src/pages/account_settings.dart

```diff
@@ -29,8 +29,6 @@ class AccountSettingsPage extends StatelessWidget {
                   AppLocalizations.of(context).accountOptionsRemoveConfirm(account.client.humanReadableID),
                 )) {
                   bloc.removeAccount(account);
-                  // ignore: use_build_context_synchronously
-                  Navigator.of(context).pop();
```

Comment:
When you have multiple accounts and remove the one that isn't currently the active one, then the account settings page of the account you just removed stays open.